MinecraftClient/Program.cs

```diff
@@ -121,22 +122,22 @@ static void Main(string[] args)
             //Test line to troubleshoot invisible colors
             if (Settings.DebugMessages)
             {
-                ConsoleIO.WriteLineFormatted("Color test: Your terminal should display [0123456789ABCDEF]: [§00§11§22§33§44§55§66§77§88§99§aA§bB§cC§dD§eE§fF§r]");
+                Translations.WriteLineFormatted("debug.color_test");

```

Comment:
Maybe keep the `[0123456789ABCDEF]: [§00§11§22§33§44§55§66§77§88§99§aA§bB§cC§dD§eE§fF§r]` part out of the translation message? I think there is no need to translate it and we may need to adjust that part later.